packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -20,6 +21,33 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     this.db = db;
     this.tableMap = buildModelTableMap(baseType);
     this.collectionName = this.tableMap.tableName;
+    Object.keys(baseType.getFields()).forEach((k: string) => {
+      const relationshipData = parseRelationshipAnnotation(baseType.getFields()[k].description)
+      if (relationshipData?.kind && ['manyToOne', 'manyToMany'].includes(relationshipData.kind)) {
+        this.db.collection(this.collectionName).createIndex({
```

Comment:
You can have ensureIndex that checks if index exist

Reply:
[ensureIndex](https://mongodb.github.io/node-mongodb-native/3.6/api/Collection.html#ensureIndex) is actually deprecated, createIndex officially replaces ensureIndex, and if index already exists, createIndex does not touch anything else